src/lib/datepicker/datepicker.spec.ts

```diff
@@ -49,7 +56,13 @@ describe('MatDatepicker', () => {
         ...imports
       ],
       providers,
-      declarations: [component],
+      declarations: declarations,
```

Comment:
I don't think we need to take `declarations` in as a separate thing, you can just do:

```ts
declarations: [component, ...entryComponents]
```

Reply:
done in https://github.com/angular/material2/pull/10856/commits/47acc75b84f8afe9142c9a043893ddae7aac4b38